localisation/wc_religions.csv

```diff
@@ -20,16 +20,16 @@ cult_of_strength;Gnoll;gnolle;Gnoll;;;;;;;;;;;x;
-burning_legion_religion;Legion;l�gion ardente;Legion;;;;;;;;;;;x;
-burning_legion_religion_DESC;The Burning Legion is a vast, innumerable army of demons, infernals, and corrupted races who seek to destroy any trace of order in the universe. They have thrice invaded Azeroth and were only beaten back at a terrible cost. Much of Azeroth's history - including the First, Second, and Third War - were due to the Legion's influence. It can be said that the Legion is Azeroth's nemesis, it corrupted the orcs who then invaded Azeroth, created the Lich King, and is the source of most of Azeroth's scars. It should also be noted that Azeroth is the only known planet that they have ever withstood the conquest.;;;;;;;;;;;;;x;
-illidari;Illidari;illidarie;Illidari;;;;;;;;;;;x;
-illidari_DESC;The Illidari are the servants of Illidan Stormrage and originally represented a specific, elite faction of his army made of demon hunters, but this has changed under the Betrayer's leadership to also include broken, naga, and shivarra forces. The Illidari are also known as Illidan's army. It consists of various factions that have banded together under Illidan's rule. They were one of the prime enemies of the Alliance and Horde in Outland, the other being the Burning Legion. Illidan commanded vast armies throughout Outland, and even a few selected groups on Azeroth.;;;;;;;;;;;;;x;
+orcish_fel_DESC;Whether its power is found through contracts between warlocks and demonic lords or through the blood of a certain Annihilan, fel threatens to corrupt completely and absolutely. Though famous for its fel-fueled crusade, the Burning Legion is not responsible for all of those who find their way to fel corruption, and those who fall on their own find themselves craving power unending, normally until their very undoing. The fel-corrupted are forever hungry for power and control, even if it should cost them their soul or the lives of their people.; those who fall on their own find themselves craving power unending, normally until their very undoing. The fel-corrupted are forever hungry for power and control, even if it should cost them their soul or the lives of their people.;;;;;;;;;;;;;x
+burning_legion_religion;Sargerite;;;;;;;;;;;;;x;
+burning_legion_religion_DESC;Bent on destroying all life in the universe, Sargeras is seen as one of the greatest threats to creation, and in a smaller sense, Azeroth. At his beck and call during his burning crusade across the Great Beyond is a loyal, fel-infused legion of demons, the Burning Legion, ready to carry out the destruction Sageras so very much desires. Followers of this dark Titan are only capable of cruelty and deceit, as they chase any chance to bring about the extinguishment of life.;;;;;;;;;;;;;x;
+illidari;Vindicatory;;;;;;;;;;;;;x;
```

Comment:
are Sargerite and Vindicatory a real thing?

Reply:
No, they're atheists in the lore.

Reply:
hmm, got no idea how to translate those...

Reply:
@arithon keep English versions?

Reply:
burning_legion_religion;Sargerite;;Sargeritisch;;;;;;;;;;;x;
burning_legion_religion_DESC;Bent on destroying all life in the universe, Sargeras is seen as one of the greatest threats to creation, and in a smaller sense, Azeroth. At his beck and call during his burning crusade across the Great Beyond is a loyal, fel-infused legion of demons, the Burning Legion, ready to carry out the destruction Sageras so very much desires. Followers of the Dark Titan are only capable of cruelty and deceit, as they chase any chance to bring about the extinguishment of life.;;Sargeras, der darauf aus ist, alles Leben im Universum zu zerstören, gilt als eine der größten Bedrohungen für die Schöpfung, und in einem kleineren Sinne, für Azeroth. Auf sein Geheiß und seinen Ruf inmitten seines brennenden Kreuzzugs durch das große Jenseits ist eine loyale, mit Fel infizierte Legion von Dämonen, die Brennende Legion, bereit, die Zerstörung durchzuführen, die Sageras so sehr begehrt. Die Anhänger des dunklen Titan sind nur zu Grausamkeit und List fähig, da sie jede Chance auf die Auslöschung des Lebens verfolgen.;;;;;;;;;;;x;
illidari;Vindicatory;;Rächend;;;;;;;;;;;x;
illidari_DESC;The powers of fel can be found tempting to all living beings in the universe, but there are those who use it as a tool to fight the evil that lurks throughout creation, rather than for personal gain or destruction. Though these individuals can often be found guilty of similar crimes to that of power-hungry warlocks and demons, they see any suffering inflicted as means to a greater end, an end greater than themselves and their sacrifices. Though more benevolent than most demons and warlocks, society will still seek to reject them and their cruel, yet effective, methods.;;Die Kräfte des Fel können für alle Lebewesen im Universum verlockend sein, aber es gibt auch diejenigen, die es als Werkzeug benutzen, um das Böse zu bekämpfen, das in der ganzen Schöpfung lauert, und nicht für persönlichen Gewinn oder Zerstörung. Obwohl diese Individuen oft ähnlicher Verbrechen schuldig sind wie machthungrige Hexenmeister und Dämonen, sehen sie jedes zugefügte Leiden als Mittel zu einem größeren Zweck, einem Zweck, der größer ist als sie selbst und ihre eigenen Opfer. Obwohl sie wohlwollender ist als die meisten Dämonen und Hexenmeister, wird die Gesellschaft trotzdem versuchen, sie und ihre grausamen, aber wirksamen Methoden abzulehnen.;;;;;;;;;;;x;